container-registry/container-analysis/pom.xml

```diff
@@ -70,13 +77,14 @@
       <artifactId>netty-tcnative-boringssl-static</artifactId>
       <version>2.0.27.Final</version>
     </dependency>
-
     <!-- [END dependencies] -->
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
       <version>4.12</version>
       <scope>test</scope>
     </dependency>
+    <!-- [START dependencies] -->
```

Comment:
Remove

Reply:
These are intentional - the snippet is showing the following:

```
<dependencyManagement>
 // bom stuff here
</dependencyManagement>

<dependencies>
  <dependency>
    // included dependency
  </dependency>

  // excluded dependencies 

</dependencies>
```

Reply:
But it's using the same tag name as above in the same file, does that work?

Reply:
@kurtisvg can you confirm this is ok to do?

Reply:
Yes - there are two ways to have "multi-part" snippets. You can use `[START_EXCLUDE]`/`[END_EXCLUDE]` or you can just end the tag and start it in the correct place (see http://go/code-snippets-201#including-code-from-multiple-sections-of-a-file)

Reply:
Neat, did not know that!